docs/server.md

```diff
@@ -107,6 +107,14 @@ There are no default required arguments to start the server
 1. **start**: optional, A more descriptive way to start the server.
 1. **stop**: optional, Stop the server if it is already running.
 
+#### Argument Priority:
+Arguments can be set in multiple locations (ex: command line, config.properties). The following is the priority:
```

Comment:
```suggestion
Arguments can be set in multiple locations (ex: command line, config.properties). The following is the priority from highest to lowest:
```

Also, should we include where environment variable configurations stand in this list?

Reply:
Environment variables take highest precedence but enable_envvars_config has to be set true in config.properties, so it isnt enabled by default. I can add it to the doc if needed.